server/graphql/v1/CollectiveInterface.js

```diff
@@ -1111,7 +1111,7 @@ const CollectiveFields = () => {
         } else if (collective.type === types.EVENT) {
           const ParentCollectiveId = collective.ParentCollectiveId;
           const parentCollective = ParentCollectiveId && (await req.loaders.Collective.byId.load(ParentCollectiveId));
-          return parentCollective && parentCollective.isApproved();
+          return parentCollective && (parentCollective.isHostAccount || parentCollective.isApproved());
```

Comment:
In the future, we should make it possible to directly read the approvedAt of the event